.circleci/config.yml

```diff
@@ -381,7 +381,7 @@ commands:
             - provision-testnet-hosts:
                 liveness-timeout-secs: 120
                 var-file: '<< parameters.var-file >>'
-                var-region: "us-east-1"
+                var-region: "us-west-2"
```

Comment:
I think we moved the region to "us-east-1" already since last week. Please make sure do we want to move back to "us-west-2" again.

Reply:
CI runs this morning failed in us-east-1. so changed to west-2. Once the PR is approved, I will change it back to east-1 before merging.